packages/features/calendars/CalendarSwitch.tsx

```diff
@@ -38,7 +38,7 @@ const CalendarSwitch = (props: ICalendarSwitchProps) => {
       const body = {
         integration: type,
         externalId: externalId,
-        ...(domainWideDelegationCredentialId && { domainWideDelegationCredentialId }),
+        domainWideDelegationCredentialId: domainWideDelegationCredentialId ?? null,
```

Comment:
Because in `/api/availability `domainWideDelegationCredentialId is required but could be null

Toggle would throw error for non-dwd credential calendar toggle otherwise